applications/app/controllers/SignupPageController.scala

```diff
@@ -15,20 +16,35 @@ class SignupPageController(
     wsClient: WSClient,
     val controllerComponents: ControllerComponents,
     csrfAddToken: CSRFAddToken,
+    groupedNewslettersAgent: GroupedNewslettersAgent,
 )(implicit context: ApplicationContext)
     extends BaseController
-    with ImplicitControllerExecutionContext {
+    with ImplicitControllerExecutionContext
+    with GuLogging {
 
   val defaultCacheDuration: Duration = 15.minutes
 
+  def logApiError(error: String): Unit = {
```

Comment:
this seems a slightly unnecessary abstraction if we're only using in one place, maybe inline it?